scipy/linalg/tests/test_lapack.py

```diff
@@ -559,3 +559,43 @@ def test_sgesdd_lwork_bug_workaround():
     assert_equal(returncode, 0,
                  "Code apparently failed: " + p.stdout.read())
 
+
+class TestSytrd(object):
+    def test_sytrd(self):
+        np.random.seed(42)
+        n = 3
+        A = np.random.normal(size=(n, n))
+        A += A.T
+
+        sytrd, sytrd_lwork = get_lapack_funcs(('sytrd', 'sytrd_lwork'))
+
+        # query lwork
+        lwork, info = sytrd_lwork(n)
+        assert info == 0
```

Comment:
Naked `assert`s  are not allowed. You can use `assert_equal` from `numpy.testing` module